server/serviceClients/authClient/index.js

```diff
@@ -1 +1,17 @@
-export * from './ucp'
+import {AuthClient as UcpAuth} from './ucp'
+import {AuthClient as SophtronAuth} from './ucp'
+import {AuthClient as LocalAuth} from './local'
+const config = require('../../config');
+
+export function CreateAuthClient(token, authProvider){
+  switch((config.AuthProvider || authProvider).toLowerCase() ){
+    case 'sophtron':
```

Comment:
Why does this sophtron case need to exist if it's functionality doesn't differ from ucp?

Reply:
they are not the same thing, they are 2 non-related providers that happens to look similar

Reply:
As Craig mentioned above it looks like the import is wrong. Right now they are the same thing, but after changing the import they won't be the same thing